doc/api/tls.md

```diff
@@ -876,6 +876,30 @@ socket.on('end', () => {
 });
 ```
 
+Note: When using an instance of `net.Socket`, you start the `net.Socket`. You can use `net.Socket` to upgrade an existing socket you don't wrap the `net.Socket` in a `TLSSocket`, then the connect works as expected.
```

Comment:
There is a grammar mistake in the second sentence: "You can use `net.Socket` to upgrade an existing socket **???** you don't wrap the..."